client/signup/controller.js

```diff
@@ -245,6 +245,18 @@ export default {
 				experiment.variationName === 'treatment_scrambled';
 		}
 
+		// See: 1113-gh-Automattic/experimentation-platform for details.
+		if ( flowName === 'guided' && userLoggedIn ) {
+			// Load both experiments in parallel for better performance.
+			const [ bigSkyExperiment, trailMapExperiment ] = await Promise.all( [
+				loadExperimentAssignment( 'explat_test_calypso_signup_onboarding_bigsky_soft_launch' ),
+				loadExperimentAssignment( 'explat_test_calypso_signup_onboarding_trailmap_guided_flow' ),
```

Comment:
We should be calling assignment in series (not parallel) and `trailmap_guided_flow ` assignments should only be called if users in `bigsky_soft_launch` are assigned the `'trailmap'` variation name. See https://github.com/Automattic/experimentation-platform/issues/1113#issuecomment-2136414776

Reply:
Yes, I did that initially! But then realized 75% of users (`trailMap` variant) will lose the performance improvements. So I sacrificed the 25% and gave them extra bytes, but not much slower performance because it's in parallel.